WeatherXM/wxm-android
# FE-1286 Support Android 15

### **Why?**
Support Android 15

### **How?**
- Updated `compileSdk` and `targetSdk`
- Updated some libraries that support Android 15
- Some minor fixes in some edge cases in Widgets that can occur in Android 15

### **Testing**
1. Install the latest SDK: https://developer.android.com/about/versions/15/setup-sdk#install-sdk
2. Setup an emulator with SDK 35 (**NOT the 16kb page size version**)
3. Test the app in the emulator and ensure everything looks OK
